index.html

```diff
@@ -95,78 +95,62 @@ <h3 class="ability_header">Skills</h3>
-                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-                    </p>
-                    <ul class="languages">
-                        <li>css</li>
-                        <li>html</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
+                    <h3></h3>
+                    <p class="work_info"></p>
+                    <ul class="languages projectOne">
+                        <li></li>
+                        <li></li>
+                        <li></li>
+                        <li></li>
                     </ul>
-                    <button type="button" class="btn">See Project</button>
+                    <button type="button" class="btn openPopUp"></button>
                 </div>
             </div>
 
             <div class="work_element work_element_reverse">
-                <img class="card" src="images/placeholder.png" alt="placeholder for pending images">
                 <div class="supporting_text">
-                    <h3>Multi-Post Stories</h3>
-                    <p class="work_info">
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.
-                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-                    </p>
-                    <ul class="languages">
-                        <li>css</li>
-                        <li>html</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
+                    <h3></h3>
+                    <p class="work_info"></p>
+                    <ul class="languages projectTwo">
+                        <li></li>
+                        <li></li>
+                        <li></li>
+                        <li></li>
                     </ul>
-                    <button type="button" class="btn">See Project</button>
+                    <button type="button" class="btn openPopUp"></button>
                 </div>
             </div>
 
             <div class="work_element">
-                <img class="card" src="images/placeholder.png" alt="placeholder for pending images">
                 <div class="supporting_text">
-                    <h3>Multi-Post Stories</h3>
-                    <p class="work_info">
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.
-                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-                    </p>
-                    <ul class="languages">
-                        <li>css</li>
-                        <li>html</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
+                    <h3></h3>
+                    <p class="work_info"></p>
```

Comment:
- [ ] Great job so far. please align your see source and see live buttons well like the figma version.
yours
![Screenshot (436)_LI](https://user-images.githubusercontent.com/65298048/188985561-e099944a-bfc6-484f-8fbc-3e2591af231c.jpg)
figma

![Screenshot (437)](https://user-images.githubusercontent.com/65298048/188985788-3515cbf0-d900-43b7-8666-de45e00ed9ac.png)

Reply:
Implemented 👍